src/app.ts

```diff
@@ -40,3 +45,76 @@ export default async function newApp(): Promise<Express> {
   }
   return app;
 }
+
+function registerRoutes(app: Express): void {
+  app.get(
+    "/public-administrations",
+    [
+      query("search")
+        .not()
+        .isEmpty()
+        .withMessage("a value is required")
+        .isLength({ min: 3 })
+        .withMessage("value must have at least 3 characters")
+    ],
+    async (req: Request, res: Response) => {
```

Comment:
async handlers are not supported in [express 4](https://github.com/expressjs/express/issues/3991) - as you can see [here](https://github.com/expressjs/express/blob/master/lib/router/layer.js#L95) Express is missing the logic to catch promise rejections

Reply:
also, declare this handler has a function, instead of inlining its body here